modules/ROOT/pages/manage-message-flows.adoc

```diff
@@ -0,0 +1,134 @@
+
+. If you want to add a message type, click *Select* under the Source message, select *New* and select a standard type such as X12 or JSON.
+
+. For a new message type, depending on the message standard, refer to the following:
++
+* xref:document-types.adoc[X12 and JSON Message Types]
+
+.   If you want to set up a target endpoint click *Select* from the Target at <partner name>.
++
+* To create a new endpoint, depending on the protocol, refer to the following:
+
+* xref:endpoint-sftp-receive-target.adoc[SFTP Receive Endpoint Settings]
+
+* xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
+
+. After you update any of the source, map, and target sections in the interface, confirm that each shows a green check mark next to it in the flow, and click *Deploy* to deploy the configurations to your Mule Runtime.
+
++
+NOTE: It may take five minutes or so for your configuration changes to sync with Mule runtime.
++
+
+. To test your edited message flow, refer to xref:configure-message-flows.adoc[Configure Message Flows].
+
+Once you confirm successful deployment of your message flow, you can track its processed messages from the Activity page.
+
+== Undeploy Message Flows
+
+You might want to undeploy a message flow so that you can delete it. Other reasons for undeploying a message flow are:
+
+* To stop processing messages related to the deployed message flow configuration.
+
+* To recover resources being consumed by a message flow that you no longer need.
+
+When you undeploy a message flow, it stops any future messages from being processed by that flow.
+Undeploying a message flow usually about five minutes. Anypoint Partner Manager checks to see if any of the applications on your Mule runtime are orphaned as a result of the undeployment.
+These applications are cleaned up to release any resources they may be consuming. You must undeploy a message flow before you delete it.
+
+Your on-premise Mule runtimes, the servers hosting the runtimes, and the `B2BserverGroup` need to be active and running when attempting an undeployment.
+If you are unsure, you should contact your B2B System Administrator or check the status of the server in Runtime manager: https://docs.mulesoft.com/runtime-manager/servers-settings[Server Settings].
+
+IMPORTANT: It may take five minutes or so for your undeployment action to complete.
+During this time, do not log out of Anypoint Partner Manager or close your browser window.
+When an undeployment is in-progress, other actions on the message flow such as `Deploy` and `Delete` are unavailable.
+
+. Make sure the server group in Anypoint Runtime Manager called: B2BServerGroup is running.
+
+. From within your Anypoint Partner Manager environment (either Sandbox or Production), click *Message Flows* in the upper left and select a message flow.
+
+. Click *Undeploy* in the upper right.
+
```

Comment:
Now add the postreq info:

During undeployment processing, do not log out of Anypoint Partner Manager or close your browser window. Anypoint Partner Manager checks to see if any of your Mule applications are orphaned as a result of the undeployment. <and then what happens?>